src/components/pages/document/show/sidebar/table-scientific-names.tsx

```diff
@@ -0,0 +1,83 @@
+// eslint-disable-next-line simple-import-sort/imports
+import { Box, Button, Skeleton, Table, Tbody, Td, Th, Thead, Tr } from "@chakra-ui/react";
+import DeleteActionButton from "@components/@core/action-buttons/delete";
+import ExternalBlueLink from "@components/@core/blue-link/external";
+import BoxHeading from "@components/@core/layout/box-heading";
+import LocalLink from "@components/@core/local-link";
+import { stickyTh } from "@components/pages/observation/list/views/stats/common";
+import useGlobalState from "@hooks/use-global-state";
+import { axUpdateScientifcNameToIsDeleted } from "@services/document.service";
+import { adminOrAuthor } from "@utils/auth";
+import useTranslation from "next-translate/useTranslation";
+import React, { useEffect, useState } from "react";
+
+export default function ScientificNamesTable({
+  data,
+  title,
+  loadMoreNames,
+  authorId,
+  refreshFunc
+}) {
+  const { t } = useTranslation();
+
+  const [showActions, setShowActions] = useState<boolean>();
+  const { isLoggedIn } = useGlobalState();
+
+  useEffect(() => {
+    setShowActions(adminOrAuthor(authorId));
+  }, [isLoggedIn]);
+
+  return data?.list?.length > 0 ? (
+    <Box className="white-box">
+      <BoxHeading>⭐ {title}</BoxHeading>
+      <Box w="full" overflowY="auto" h={360}>
+        <Table variant="striped" colorScheme="gray" size="sm">
+          <Thead>
+            <Tr>
+              <Th {...stickyTh}>{t("document.show.scientific_names_table.name_header")}</Th>
```

Comment:
it should be `document:` instead of `document.` everywhere

Reply:
got it, this was causing the issue, it was just displaying the keys in the UI. I missed the new system that we use now in the codebase

Reply:
Correction added.